modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -112,27 +85,92 @@ export default {
   },
 
   computed: {
+    moduleOptions() {
+      return window.apos.modules[this.moduleName] || {};
+    },
+    schema() {
+      return this.moduleOptions.schema || [];
+    },
+    groups() {
+      const groupSet = {
+        basics: {
+          label: 'Basics',
```

Comment:
Making sure everything is in a fallback group by the end of "compose" should be the server's problem because it's easier to configure, i18n and change things there. But, maybe a fallback here can't hurt.

Reply:
True. Though if I'm treating this as the fallback it should be "Other".

Reply:
Maybe this is a bad idea. We allow people to set their own `defaultGroup`.